code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -1530,7 +1531,7 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	name = "Quintuple Sec"
 	description = "Law, Order, Alcohol, and Police Brutality distilled into one single elixir of JUSTICE."
 	color = "#ff3300"
-	boozepwr = 80
+	boozepwr = 150
```

Comment:
hot DAMN 150% alcohol

Reply:
it heals five of each damage type per tick

Reply:
Also requires clown tear soup ( which needs bananium), which is extremely rare iirc. Good luck getting that to be used for a drink

Reply:
> Also requires clown tear soup ( which needs bananium), which is extremely rare iirc. Good luck getting that to be used for a drink

not like bananium can be avaliable roundstart

Reply:
> > Also requires clown tear soup ( which needs bananium), which is extremely rare iirc. Good luck getting that to be used for a drink
> 
> not like bananium can be avaliable roundstart

Why you doubling down on this lol, its very rare for it to be roundstart and even then. Why would anyone that finds it waste it on a drink for sec? Ignoring round start, Quinti is one of the "hardest" drinks to make due to how rare the bananium resource is